src/sql_queries.py

```diff
@@ -0,0 +1,20 @@
+query_labelled_feedback = """
+
+SELECT
+    feedback.type,
+    feedback.created,
```

Comment:
Would it be easier to change this to DATE(feedback.created) so you don't have to adjust for datetime in other files? 

Same with CAST(feedback.feedback_record_id AS STRING)?

Reply:
Agree, will change